go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/GoogleContainerTools/kpt
 
 go 1.17
 
+replace github.com/GoogleContainerTools/kpt/porch/api => ./porch/api
```

Comment:
The current CLI is incompatible with the changes I made on the backend. My temporary solution is to make the needed changes on the CLI side and pin it to the local porch/api directory so that the CLI tests can pass, and then remove this replace statement immediately in a followup PR when this merges.